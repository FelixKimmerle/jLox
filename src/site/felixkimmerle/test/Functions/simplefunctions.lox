fun test(a,b,c)
{
    return a+b*c;
}

print test;
print test(1,2,3);
print test(1,2);