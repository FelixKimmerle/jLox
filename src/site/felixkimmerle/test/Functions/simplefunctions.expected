<fn test>
7
Expected 3 arguments but got 2.
[line 8]
